src/lib/formats/dmk_dsk.cpp

```diff
@@ -83,6 +81,27 @@ int dmk_format::identify(util::random_read &io, uint32_t form_factor, const std:
 }
 
 
+uint32_t dmk_format::wide_fm(uint32_t val)
```

Comment:
It seems this `val` parameter could be `uint16_t`.

Reply:
Changed.